client/src/ui/deckbuilder/DeckBuilderColumn.vue

```diff
@@ -1,12 +1,13 @@
 <template>
   <div
       class="_column"
-      :class="{columnDrop: dropTargetIndex !== null}"
+      :class="{columnDrop: dropTargetIndex !== null,
```

Comment:
Nit:

```
:class="{
  columnDrop: dropTargetIndex !== null,
  horizontal,
}"
```

(There's nothing wrong with the original form, but it's hard to align the secondary lines without an autoformatter. Alternately, do you want to stand up an autoformatter for the JS/Vue files? :D)